apps/bluetooth-music/app.js

```diff
@@ -52,16 +47,14 @@ function onAudioFocusLost (transient, mayDuck) {
   }
 }
 
-function uploadEvent (event, data) {
+function uploadEvent (event) {
   logger.debug('upload bluetooth event', event)
-  var MEDIA_SOURCE_BLUETOOTH = 3
-  var event2status = {
-    PLAYING: 0,
-    STOPPED: 1,
-    PAUSED: 2
-  }
-  var msg = [ MEDIA_SOURCE_BLUETOOTH, event2status[event] ]
-  agent.post('yodaos.apps.multimedia.playback-status', msg)
+  agent.post('yodaos.apps.bluetooth.playback-status', event)
```

Comment:
Suggest the `yodaos.apps.bluetooth.multimedia.playback-status` to keep consistent with the cloud-player.

Reply:
modified.